.github/workflows/test_deploy.yml

```diff
@@ -558,18 +558,21 @@ jobs:
         name: release-url
         path: release-url.txt
 
-  push-image:
+  build-images:
     runs-on: ubuntu-latest
-    needs: [docs-linux, test-linux-cli, test-linux-core, test-linux-service, test-macos-cli, test-macos-core, test-macos-service]
-    if: "startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'"
+    # needs: [docs-linux, test-linux-cli, test-linux-core, test-linux-service, test-macos-cli, test-macos-core, test-macos-service]
     steps:
     - uses: actions/checkout@v2
+    - name: build images
+      run: make docker-tag
     - name: Docker Login
+      if: "startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'"
```

Comment:
nitpicking: in case we change the default branch from `master` to `main` or something else this requires a patch